qemu/tests/cgroup.py

```diff
@@ -602,8 +604,8 @@ def _test(direction, blkio):
         # ; true is necessarily when there is no dd present at the time
         kill_cmd = "rm -f /tmp/cgroup_lock; killall -9 dd; true"
         stat_cmd = "killall -SIGUSR1 dd; true"
-        re_dd = (r'(\d+) bytes \(\d+\.*\d* \w*\) copied, (\d+\.*\d*) s, '
-                 '\d+\.*\d* \w./s')
+        re_dd = (r'(\d+) bytes \(\d+\.*\d* \w*(, \d+\.*\d* \w*)?\) copied, '
```

Comment:
I couldn't properly follow this change (unless I spent too much time running and debugging this code).  Can you explain why this changed with the `shell=True` change?

Reply:
I just forgot to separate this change. Newer `dd` versions have different format. Do you want me to split these patches or can it be part of it?

Reply:
OK, I'm splitting in v3 and omitting in the push